### DISCUSSION TITLE
Well there we go…

### DISCUSSION THREAD

**Discussion by Mreco99** [actualbudget/releases#253] [Feature ideas]
I found actual budget , spent 3weeks setting it up and getting used to it , literally subscribed two days ago , and now it’s gone to open source. 
it doesn’t give me confidence that I should carry on with AB. I’m staggered that there were only 810 subscribers. I would have thought there were over 10k subscribers. This is a great bit of software for me! I don’t understand half of what he final email said

**Comment by cristianbostan** [comment]
I understand what you're saying and i get your frustration. I was "afraid" of this from the beginning, but the fact that it went open source isn't necessarily a bad thing. It would've been much worse if the project was simply abandoned. Now any other brilliant developer (and there are a lot of them out there) can pick it up and carry it forward. The project isn't dead and if people will see its potential, it will definitely grow.
I've tried several other budgeting solutions before starting using Actual and I think it is superior to many of them. Just wait a little bit until some of the details of self-hosting for less technical users get ironed out. It's not that complicated and it works almost in the same way.

**Reply by MikeBishop** [reply]
Agreed.  I'm far more concerned about the mobile apps than the self-hosting.  Running one more container on my NAS is no big deal; sideloading an Android app is a harder sell, particularly on my wife's phone.
